src/ui/public/state_management/state.js

```diff
@@ -205,7 +205,11 @@ export function StateProvider(Private, $rootScope, $location, config, kbnUrl) {
    */
   State.prototype._parseQueryParamValue = function (queryParam) {
     if (!isStateHash(queryParam)) {
-      return rison.decode(queryParam);
+      try {
+        return rison.decode(queryParam);
+      } catch(err) {
+        return null;
+      }
```

Comment:
I would rather bubble the errors if the session storage is getting corrupted somehow